Hack to avoid dependency on obsolete ed25519 package

cajun-rat opened the pull request.

cajun-rat committed:
Hack to avoid dependency on obsolete ed25519 package

tkfu reviewed (approved):
LGTM

tkfu commented:
Ah, CI pipeline failed. Looks like it's because the pipeline is ultimately installing the venv from requirements-dev.{in,txt} Can you change those to use pynacl as well?

tkfu reviewed (changes requested):
Changes also need to be done in requirements-dev.in and requirements-dev.txt

cajun-rat committed:
Fixes to get the CI back running

This can be tested locally with:

    docker build -t tuf-test-vectors-ci -f ci/gitlab/Dockerfile.tuf-test-vectors  .
    docker run -v "$PWD:$PWD" -w "$PWD" -it tuf-test-vectors-ci make lint test

cajun-rat commented:
@tkfu It turns out to be quite an adventure to fix this. Hopefully this fixes it.  The CI doesn't run automatically since I'm not a maintainer for this. Any chance you could add me?
